lib/date/src/root.zig

```diff
@@ -0,0 +1,9 @@
+const std = @import("std");
+const testing = std.testing;
+
+pub usingnamespace @import("parsing.zig");
```

Comment:
usingnamespace is planned to be removed from zig https://github.com/ziglang/zig/issues/20663

Reply:
True, I am aware of it. But it is oh so convenient. 

I'm sort of hoping that they renege on removing it from the language. 

However, I will update the code to not use it. The easiest way would be to make `date.zig` the root source file, but I wonder if I can reinvent `usingnamespace` with comptime.

Reply:
Updated 040a1f1d00624c2435d63857c4e23702bba94b3b